pkg/openstack/loadbalancer.go

```diff
@@ -1910,6 +1910,10 @@ func (lbaas *LbaasV2) ensureOctaviaLoadBalancer(ctx context.Context, clusterName
 
 // EnsureLoadBalancer creates a new load balancer or updates the existing one.
 func (lbaas *LbaasV2) EnsureLoadBalancer(ctx context.Context, clusterName string, apiService *corev1.Service, nodes []*corev1.Node) (*corev1.LoadBalancerStatus, error) {
+	if !lbaas.opts.Enabled {
+		return nil, cloudprovider.ImplementedElsewhere
+	}
+
```

Comment:
Did I understand correctly from the PR description that we can't either:
* Stop watching Service
* Entirely disable the Service controller

Because it may need to Reconcile on delete? Is that correct? Is there anything else we still need to reconcile in CPO?

Reply:
We don't have any control over Service controller here as it's running in kubernetes/cloud-provider and here we only implement an interface that's called there. There is an explanation on how it's supposed to work here: https://github.com/kubernetes/cloud-provider/blob/master/cloud.go#L117-L132

This seems to makes sense - imagine someone running with `enabled: true`, creating a few LBs and then switching to `enabled: false`. It's still expected that when these Services will get deleted, the Octavia LB will get removed, even though the integration is disabled now.

Reply:
@mdbooth the other resource creation is in other controllers, here we only create LB for LB typed services , so should be ok to return here directly